scripts/copyDTSFiles.js

```diff
@@ -1,22 +1,26 @@
-const path = require('path');
+#!/usr/bin/env node
+
+/* eslint-disable import/no-extraneous-dependencies, no-console */
+const { copy } = require('fs-extra');
 const glob = require('glob');
-const fse = require('fs-extra');
+const path = require('path');
 
-// copies all .d.ts files to lib/ directory
-// inspired by https://github.com/mui-org/material-ui/blob/v1-beta/scripts/copy-files.js
+// copies all .d.ts files to esm/ directory
+// This can go away once there are no explicit
```

Comment:
Not sure if this comment is complete 🤔

Reply:
It is not complete! Fixing now.
The intent is that this operation can go away when everything is converted to TS and we have no more explicit .d.ts files.